Framework/DataHandling/src/SaveTBL.cpp

```diff
@@ -83,16 +83,28 @@ void SaveTBL::exec() {
         std::string groupHeading = columnHeadings[columnIndex];
         if (ws->getColumn(groupHeading)->type() != "int") {
           file.close();
-          remove(filename.c_str());
-          throw std::runtime_error(groupHeading +
-                                   " Column must be of type \"int\"");
+          int error = remove(filename.c_str());
+          if (error == 0)
```

Comment:
I've just reread that and I didn't take the error throwing in context with line 84 so I see now that the error relates to the column type being incorrect.

Please ignore my previous comment.